.github/workflows/pipeline.yaml

```diff
@@ -0,0 +1,68 @@
+name: IBM Volume File VPC
+
+on:
+  push:
+    branches:
+      - master   # commit pushed directly to the master
+    tags:
+      - "v[1-9]+.[0-9]+.[0-9]+*"   # tag creation
+  pull_request:
+    branches:
+      - master   # trigger workflow when opened a PR into master.
+
+# Cancel older runs on the same pull request
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # needed to push to gh-pages
+      pull-requests: write # needed to comment on PRs
+    strategy:
+      matrix:
+        go-version: ["1.25.3", "stable"]
+      fail-fast: false
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go-version }}
+          check-latest: true
+
+      - name: Install Go dependencies
+        run: make deps
+
+      - name: Run make fmt
+        run: make fmt
+
+      - name: Set up Go workspace
+        run: |
+          mkdir -p $(go env GOPATH)/src/github.com/IBM/ibmcloud-volume-file-vpc
+          rsync -az . $(go env GOPATH)/src/github.com/IBM/ibmcloud-volume-file-vpc
+        env:
+          GOPATH: ${{ runner.temp }}/go
+
+      - name: Run go vet
+        run: make vet
+
+      - name: Run tests
+        run: make test
+
+      - name: Generate coverage
+        run: make coverage && touch "Passing" || touch "Failed"
+
+      - name: Publish coverage
+        if: success() && matrix.go-version == 'stable'
+        env:
+          GHE_TOKEN: ${{ secrets.GHE_TOKEN }}
```

Comment:
I am bit concern about using this TOKEN, can be at least disable tracing, are we disabling global tracing by using `set +x`